package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "treetracker",
-  "version": "1.38.4",
+  "version": "1.38.0",
```

Comment:
We shouldn't change and push this file, changing version number is automatically down by the semantic release